From 029da8f9485ebee7be224fa3d60733cff5578328 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Sat, 5 Aug 2023 10:39:28 -0500
Subject: EDAC/i10nm: Add Intel Grand Ridge micro-server support

The Grand Ridge CPU model uses same memory controller registers with
Granite Rapids server. Add Grand Ridge CPU model ID for EDAC support.

[Ricardo: Used the Grand Gridge definition from intel-family.h]

Signed-off-by: Qiuxu Zhuo <qiuxu.zhuo@intel.com>
Back-ported-by: John Jacques <john.jacques@intel.com>
---
 drivers/edac/i10nm_base.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/edac/i10nm_base.c b/drivers/edac/i10nm_base.c
index ba64926bad97..93a6e7b01626 100644
--- a/drivers/edac/i10nm_base.c
+++ b/drivers/edac/i10nm_base.c
@@ -762,6 +762,7 @@ static const struct x86_cpu_id i10nm_cpuids[] = {
 	X86_MATCH_INTEL_FAM6_MODEL_STEPPINGS(ICELAKE_D,		X86_STEPPINGS(0x0, 0xf), &i10nm_cfg1),
 	X86_MATCH_INTEL_FAM6_MODEL_STEPPINGS(SAPPHIRERAPIDS_X,	X86_STEPPINGS(0x0, 0xf), &spr_cfg),
 	X86_MATCH_INTEL_FAM6_MODEL_STEPPINGS(GRANITERAPIDS_X,	X86_STEPPINGS(0x0, 0xf), &gnr_cfg),
+	X86_MATCH_INTEL_FAM6_MODEL_STEPPINGS(GRANDRIDGE,        X86_STEPPINGS(0x0, 0xf), &gnr_cfg),
 	{}
 };
 MODULE_DEVICE_TABLE(x86cpu, i10nm_cpuids);
-- 
2.25.1

